tests/serverpod_test_server/test_e2e/service_protocol_test.dart

```diff
@@ -430,17 +430,19 @@ void main() {
             .map((c) => c.dartType)
             .toList();
 
-        expect(columns, hasLength(7));
         expect(
             columns,
-            containsAll([
+            equals([
```

Comment:
Unless I'm missing something this is equal but shorter.

Reply:
Equals also puts restrictions on the order of the items in the list while containsAll just validates that each entry is found. After this change, this test would fail if the order of the elements changes.

Since we are fetching something from the database I would assume that order is unpredictable and therefore should be excluded.

Reply:
Ah right, makes sense then 👍  Thanks!